test/core/binary.wast

```diff
@@ -37,6 +34,9 @@
 (assert_malformed (module binary "\ef\bb\bf\00asm\01\00\00\00") "magic header not detected")
 
 ;; Unknown binary version.
```

Comment:
Nit: can change this heading to "malformed binary version"?

Reply:
Yeah, it‘s more explicit, thanks!